CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## v 1.0.2 (Jan 01, 2019)
+
+* Create a callback signature from Indacoin.
```

Comment:
Let's make the sentence more precise: "Add a function that generates a callback signature"